packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -523,22 +511,26 @@ Router = Router.extend({
    * @returns boolean
    * */
   showReactApp(routeName) {
-    for (const routeGroup in this.reactRouteGroups) {
+    const reactRouteGroups = this.getReactRouteGroups();
+    for (const routeGroup in reactRouteGroups) {
       if (
-        this.reactRouteGroups[routeGroup].routes.find(
-          (route) => routeName === route
-        )
+        reactRouteGroups[routeGroup].routes.find((route) => routeName === route)
       ) {
         return (
-          this.reactRouteGroups[routeGroup].featureFlagOn &&
+          reactRouteGroups[routeGroup].featureFlagOn &&
           (this.isInReactExperiment() ||
-            ALWAYS_SHOWN_REACT_GROUPS.includes(routeGroup))
+            reactRouteGroups[routeGroup].fullProdRollout === true)
         );
       }
     }
     return false;
   },
 
+  getReactRouteGroups() {
+    const routeGroups = getClientReactRouteGroups(this.config.showReactApp);
+    return routeGroups;
```

Comment:
Not sure if it matters, but should this just return the routes directly instead of dealing w/ the temp variable? Or was the temp variable for debuggin' purposes?